romeo/src/state.rs

```diff
@@ -12,14 +12,18 @@ use sbtc_core::operations::{
 	op_return, op_return::withdrawal_request::WithdrawalRequestData,
 };
 use stacks_core::codec::Codec;
-use tracing::info;
+use tracing::{debug, info};
 
 use crate::{
 	config::Config,
 	event::{Event, TransactionStatus},
 	task::Task,
 };
 
+/// The delay in blocks between receiving a deposit request and creating
+/// the deposit transaction.
+const MINT_TRANSACTION_DELAY_BLOCKS: u32 = 3;
```

Comment:
It should be 1. The experience would be bad if we wait always 5 blocks for no reasons.

Reply:
We just need to wait until the stacks blockchain sees the bitcoin block. That will be in the next stacks block the latest.

Reply:
If this is set to 1, would it make sure we can handle 5 flash blocks in a row? I'm not sure how common that is.

Reply:
The height is the stacks height, therefore, we can handle as many flash blocks as there are.
Btc block #100 and Stx block #10 are published.
Then Btc block #101 is publish with deposit request, we wait for STX block #11 before broadcasting minting because btc block #101 is not visible while building block #11. 

(Maybe my numbers are off by 1 but we don't have to worry about the number of flash blocks. When we wait for block #11 all the btc block #101, 102, 103,... would be covered in the same way)